src/Ledger/Conway/Conformance/Ratify.lagda

```diff
@@ -272,18 +274,35 @@ actualVotes Γ pparams cc ga votes
     λ where
 \end{code}
 \begin{code}
-      (true , just (just c'))  → just c'
-      _                        → nothing -- expired, no hot key or resigned
-
-  actualCCVote : Credential → Epoch → Vote
-  actualCCVote c e = case getCCHotCred (c , e) of
+    (true , just (just c'))  → just c'
+    _                        → nothing -- expired, no hot key or resigned
 \end{code}
 \begin{code}[hide]
-    λ where
+  getPoolParams : Credential → Maybe PoolParams
+  getPoolParams c = case lookupᵐ? stakeDelegs c of λ where  nothing → nothing
```

Comment:
This lookup is incorrect, you just need to lookup `c` in `pools`.

Reply:
That's good to know, thanks.  If there is a particular place in a prior version of the spec or a CIP that would have helped me understand this better so that I wouldn't have made this mistake, please advise.

Reply:
Not really. It's just that `lookupᵐ? pools c` is the `PoolParams` associated to `c` and contains `c`'s `RewardAddr`,  which is what you want. As the code is written here, it gives you the `PoolParams` of the pool that `c` is delegating to which could be any random pool.

Reply:
I see.  Thanks for the explanation.  Very helpful. 👍🏼